extension/intents/tabs/tabs.toml

```diff
@@ -1,31 +1,45 @@
 [tabs.close]
 description = "Closes the current tab"
 match = """
-  close (the | this |) (active | current |) (tab{s} | site | page) (for me |)
+  close (the | this |) (active | current |) (tab | site | page) (for me |)
   close (the | this | that)
 """
 
 [[tabs.close.example]]
 phrase = "close tab"
 
 [[tabs.close.example]]
-phrase = "close tabs"
+phrase = "close active tab"
 test = true
 
 [[tabs.close.example]]
-phrase = "close all tabs"
+phrase = "close the active tab"
 test = true
 
 [[tabs.close.example]]
-phrase = "close active tab"
+phrase = "close the current top"
 test = true
 
-[[tabs.close.example]]
-phrase = "close the active tabs"
+[tabs.closeAll]
+description = "Closes all open tabs"
+match = """
+  close (all |) (active | open |) (tab{s} | site{s} | page{s}) (for me |)
+"""
+
+[[tabs.closeAll.example]]
+phrase = "close tabs"
```

Comment:
This is an example where "all" should be required, something like "close tab" (which is also matched for the phrase) should only close the active tab, definitely not all tabs!

Reply:
done!